### DISCUSSION TITLE
Copy current url button

### DISCUSSION THREAD

**Discussion by thewh1teagle** [bromite/bromite#1425] [Ideas]
Can we add copy current url button in the search bar like chrome for android has?
![Screenshot_20210928-134747](https://user-images.githubusercontent.com/61390950/135074511-e2186d39-2cd6-43c1-b7c1-4f73b29d89a1.png)

**Comment by suicideboy98** [comment]
+1 it's very helpful. for now u can copy by tapping lock icon and then tap and hold url
